.gitattributes

```diff
@@ -1,7 +1,7 @@
 ###############################################################################
 # Set default behavior to automatically normalize line endings.
 ###############################################################################
-* text=auto
+* text=auto eol=lf
```

Comment:
On all modern OSes, modern editors (including VS Code) can deal with `lf` endings just fine, `crlf` is not needed.
If `git` is allowed to set the line endings to `crlf` before commits, this leads to discrepancies that are annoyingly useless at best.
And who knows what at worse ! Some of these files end up being used in a Docker environment, which will be expecting `lf` endings.